README.md

```diff
@@ -1,3 +1,15 @@
 # marta-ga-us
 
-Information about MARTA (Atlanta, GA - http://www.itsmarta.com/).
+Tracking issues for [MARTA](http://www.itsmarta.com/) (Atlanta, GA) GTFS and GTFS-realtime data.
+
+### Problem with MARTA data?
+
+Look at the [existing issues](https://github.com/General-Transit-Feed-Specification/marta-ga-us/issues) first to see if someone has already reported it.
+
+If no one has, [open a new issue](https://github.com/General-Transit-Feed-Specification/marta-ga-us/issues/new).
+
+### Helpful Links
+
+* [MARTA website](http://www.itsmarta.com/)
+* [MARTA on Transitland](https://transit.land/feed-registry/operators/o-dnh-metropolitanatlantarapidtransitauthority)
+* [MARTA on TransitFeeds.com](http://transitfeeds.com/p/marta)
```

Comment:
I would add a link to the Wikipedia page because it is really useful

`* [MARTA on Wikipedia](https://en.wikipedia.org/wiki/Metropolitan_Atlanta_Rapid_Transit_Authority)`